protocol/xmrmaker/errors.go

```diff
@@ -30,6 +30,7 @@ var (
 	errRelayingWithNonEthAsset       = errors.New("relayers with ERC20 token swaps are not currently supported")
 
 	// protocol initiation errors
+	errSwapDoesNotExist          = errors.New("swap with given ID in contract does not exist")
```

Comment:
I'm in favor of standardized naming since we have several ID values:
OfferID (or Offer ID)
SwapID (or Swap ID)
PeerID (or Peer ID)

In this case, you can extract that it is the SwapID, because we don't have Offer and Peer IDs in the contract, but with standardized naming, the reader has to think less.

Reply:
you mean naming the contract swap ID `SwapID`? I'd rather name it something more explicit like `OnChainID` or `ContractSwapID` (this is usually used in places anyways)

I think it's a bit out of scope for this PR but we can open an issue for it!

Reply:
No, I was proposing that we be more rigorous in the order of the words so readers don't have to carefully parse sentences and so they can search the code base for keywords and word sequences. While I do like the phrase `OnChainID`, the code base is currently using the phrase "Swap ID" and sometimes "Contract Swap ID" to refer to the on-chain ID.  The error message can be rewritten from `swap with given ID in contract does not exist` to `contract swap ID does not exist`.

Reply:
ah, i see. okay i'll update the message!